packages/sdk/contractkit/src/wrappers/Governance.test.ts

```diff
@@ -76,16 +76,21 @@ testWithGanache('Governance Wrapper', (web3: Web3) => {
       [CeloContract.Random, '0x0000000000000000000000000000000000000001'],
       [CeloContract.Escrow, '0x0000000000000000000000000000000000000002'],
     ]
-    const proposalID = new BigNumber(1)
+    const proposalID = new BigNumber(2)
```

Comment:
why a different id?

Reply:
Thanks for this question, after bit of thinking I decided to update the tests a bit and revert this part.

Please check tests now and let me know what you think

Reply:
In general idea behind this PR is to be able to propose and approve new proposals without the need to wait any period of time unless there was other proposal dequeued previously in defined period of time. 

New proposal can be dequeued immediately which means that you will not really be able to upvote it. Some tests were relaying on the previous behaviour (required waiting time for all new proposals). Easy way how to fix those tests was to propose one more proposal (that is the reason for id change), but in the end I decided to update tests in a way that they all will not call upvote proposal and rather update "upvote" tests with be queuing one more proposal only in those tests.

Reply:
hmm im instinct here would be to say that there should be a new test, rather than change one unless the old way just wont work after the code change.

Reply:
` idea behind this PR is to be able to propose and approve new proposals without the need to wait any period of time` this seems more significant than just not update deque timestamp. 

Then again this nt the area im most familiar with